colossalai/inference/core/request_handler.py

```diff
@@ -39,10 +47,23 @@ def abort_sequence(self, seq_id: str):
         self._find_sequence(seq_id)
         return
 
-    def _find_sequence(self, seq_id: str) -> "Reqseq":
+    def _find_sequence(self, seq_id: str) -> "Sequence":
         """
         Find the request by seq_id.
         """
 
     def check_unfinished_seqs(self) -> bool:
         return self.waiting_list or self.running_list
+
+    def update(self):
```

Comment:
`check_unfinished_seqs` is not correct to me, the return type is expected to be a boolean, but a list will be returned instead.